src/components/button/button.stories.tsx

```diff
@@ -0,0 +1,76 @@
+import React from "react"
+import { storiesOf } from "@storybook/react"
+import { action } from "@storybook/addon-actions"
+import { text, boolean } from "@storybook/addon-knobs"
+import { readmeCleanup } from "../../../utils/readme"
+import { Button } from "."
+
+// @ts-ignore
+import readme from "./README.md"
+import { ButtonType } from "./button"
+import { Icon } from "../icon"
+
+const mockStory = storiesOf("Button", module)
```

Comment:
why `mockStory` it is't highly important, but anyway

Reply:
Ooops missed that...
Corrected done.